experiment_manager.py

```diff
@@ -432,8 +454,12 @@ def labber_read(self,config, labber_log_name=None):
         # print(step_list)
 
         for index, job in enumerate(self.sweep_jobs):
-            print(self.sweep_configs[index].param_list) #TODO only pring sweep parameters
+            print(f'reading result from job {index+1} out of {len(self.sweep_jobs)}...')
+            tic()
             result = self.get_observables_1D(self.sweep_configs[index], job)
+            print('1D observables read in:')
+            toc()
+            print('')
             labber_trace = result["labber_trace"]
```

Comment:
No need for that variable.
Just return `result['labber_trace']`

Reply:
fixed